### PR TITLE
[review] mtx2csv script

### PR DESCRIPTION
**Pull request by afender**

### CODE REVIEW

File: python/utils/mtx2csv.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import argparse
+
+
+parser = argparse.ArgumentParser(description='Convert the sparsity pattern \
+                                 of a MatrixMarket file into a CSV file. \
+                                 Each directed edge is explicitely stored, \
+                                 edges are unsorted, IDs are 0-based.')
+parser.add_argument('file', type=str,
+                    help='Path to the MatrixMarket file')
+parser.add_argument('--csv_separator_name', type=str, default="space",
```
Comment (raydouglass) [inline python/utils/mtx2csv.py]: You can add another parameter `choices=["space","tab","comma"]` and the parser will enforce it.

File: python/utils/mtx2csv.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import time
+from scipy.io import mmread
+import argparse
+
+
+parser = argparse.ArgumentParser(description='Convert the sparsity pattern \
+                                 of a MatrixMarket file into a CSV file. \
+                                 Each directed edge is explicitely stored, \
+                                 edges are unsorted, IDs are 0-based.')
+parser.add_argument('file', type=str,
```
Comment (raydouglass) [inline python/utils/mtx2csv.py]: If you use `type=argparse.FileType()` then the `file` arg will be a file-like object. And the parser will check that it exists and is readable, plus support using `-` for bash piping.

File: python/utils/mtx2csv.py (hunk 1 above)
Comment (afender) [reply python/utils/mtx2csv.py]: addressed in f3a16b4

File: python/utils/mtx2csv.py (hunk 2 above)
Comment (afender) [reply python/utils/mtx2csv.py]: addressed in f3a16b4

**Pull request merged**